src/mdo/reader-stax.vm

```diff
@@ -246,51 +246,76 @@ public class ${className} {
 #end
     } //-- ${root.name} read(InputStream, boolean)
 
-    /**
-     * Method read.
-     *
-     * @param parser a parser object.
-     * @param strict a strict object.
-     * @throws XMLStreamException XMLStreamException if
-     * any.
-     * @return ${root.name}
-     */
+/**
```

Comment:
Fix the formatting of this file, this completely breaks the format of the generated java file.